### ISSUE TITLE
AC Voltage off

### ISSUE THREAD

**Issue by shingle01** [David00/rpi-power-monitor#96]
This might be a case of but the pre-assembled kit stupid, but living in Australia I found a self build more economical.

I have the 0.2 image rpi image installed.

9VAC supply attached, no CT clamps connected.  I've added the initial mains and transformer voltages to the config and ran:` pip install .`

No AC voltage correction value yet but the readings are 10% off and wildly erratic vs the kill'o'watt output.

![image](https://user-images.githubusercontent.com/43157345/226284404-cb8661a4-e597-406c-a13b-a8d4d6af11c4.png)

No problems with the 3.3VDC supply.  The Voltage on PIN5 of ADC is a steady 1.89VDC.

Below is a debug output.  Strangely, despite CT1 being discounted I have a trend.

![image](https://user-images.githubusercontent.com/43157345/226284701-fc4ac9d3-3e4f-48db-9e72-77bbea24d8d6.png)

![20230320_192635](https://user-images.githubusercontent.com/43157345/226285392-876019df-deb1-46e8-b5c4-58379113db48.jpg)

When AC supply is disconnected:
![image](https://user-images.githubusercontent.com/43157345/226285702-bbb6a506-d7e2-4495-bd65-442f08adb83c.png)

**Comment by David00** [comment]
Hi @shingle01, I can tell from the debug graph that something isn't quite right with the bias_v output because all of the channels should be "idling" right around 511 in the plot.  Also, the trace that is showing up for CT1 looks very similar to a capacitor charge/discharge cycle.

I would suggest double-checking everything in the "Bias Voltage Reference" part of [the schematic](https://raw.githubusercontent.com/David00/rpi-power-monitor/docs/_includes/resources/Schematics/latest.png), and confirming the value (and capacitor type) for the filtering capacitors C0 - C3, C4, C5,

**Comment by shingle01** [comment]
Thanks for the Info.

I replaced C5 with a polarized cap (store originally gave me non-polarized).  No Luck.

Added some more solder to the legs of U1, FIXED!

Thanks for the help.

![image](https://user-images.githubusercontent.com/43157345/226833962-3d589c28-cb49-489e-9829-a1a44265bac7.png)

![image](https://user-images.githubusercontent.com/43157345/226834190-8dd8ee93-351f-4677-a0b4-4f9f2bcae151.png)
